.changesets/feat_bnjjj_add_studio_trace_id_selector.md

```diff
@@ -0,0 +1,18 @@
+### Add support for studio trace id in selectors and document context entry ([Issue #3803](https://github.com/apollographql/router/issues/3803)), ([Issue #5172](https://github.com/apollographql/router/issues/5172))
+
+Add support for a new trace id selector kind, the apollo trace id which represents the trace id on [Apollo Studio](https://studio.apollographql.com/). 
+
+Example:
```

Comment:
Do separate issues require separate changelog entries? Can edit in the eventual release's changelog.

```suggestion
### Add support for studio trace id in selectors and document context entry ([Issue #3803](https://github.com/apollographql/router/issues/3803)), ([Issue #5172](https://github.com/apollographql/router/issues/5172))

Add support for a new trace ID selector kind, the `apollo` trace ID, which represents the trace ID on [Apollo GraphOS Studio](https://studio.apollographql.com/). 

An example configuration using `trace_id: apollo`:
```